src/OrchardCore.Modules/OrchardCore.ContentFields/Views/UserPickerField.cshtml

```diff
@@ -0,0 +1,32 @@
+@model OrchardCore.ContentFields.ViewModels.DisplayUserPickerFieldViewModel
+@using OrchardCore.Mvc.Utilities
+@using OrchardCore.Users.Models
+@using OrchardCore.Users.Indexes
+@using YesSql.Services;
+@using YesSql;
+
+@inject YesSql.ISession Session
+
+@{
+    var name = (Model.PartFieldDefinition.PartDefinition.Name + "-" + Model.PartFieldDefinition.Name).HtmlClassify();
+    var users = await Session.Query<User, UserIndex>().Where(x => x.UserId.IsIn(Model.UserIds)).ListAsync();
+}
```

Comment:
Should we expose this as a `Session.Query` ?

Or extend the `IUserService` to have a `GetMany` method.

Or be more like the media field, and just provide the user id in the view?

For liquid that will mean writing a SQL / Lucene query, or should we expose part of the `IUserService`

Reply:
helper+ filter to get a user from userid